client/client.py

```diff
@@ -131,7 +134,11 @@ def login_user(self):
         }
         url = 'https://www.skotos.net/user/login.php'
         response = requests.post(url, headers=header, data=data, allow_redirects=False)
-        # pprint(response.headers)
-        self.uname = re.search('user=(.*?);', response.headers['set-cookie']).group(1)
-        self.pwd = re.search('pass=(.*?);', response.headers['set-cookie']).group(1)
+        try:
+            pprint(response.headers)
```

Comment:
I think this might have been a bad merge from some old code? Agreed, we don't really want to print in general, we should just use logs.